core/pom.xml

```diff
@@ -51,11 +52,6 @@
       <artifactId>commons-io</artifactId>
       <version>2.11.0</version>
     </dependency>
-    <dependency>
-      <groupId>commons-logging</groupId>
-      <artifactId>commons-logging</artifactId>
-      <version>1.2</version>
-    </dependency>
```

Comment:
Not imported directly, but a transitive dependency of several modules that causes Enforcer errors, so moved to the `<dependencyManagement>` section of `stapler-parent`.